internal/action/sync.go

```diff
@@ -161,10 +162,12 @@ func (s *Action) sync(ctx context.Context, store string) error {
 func (s *Action) syncMount(ctx context.Context, mp string) error {
 	// using GetM here to get the value for this mount, it might be different
 	// than the global value
-	if as := s.cfg.GetM(mp, "core.autosync"); as == "false" {
-		debug.Log("not syncing %s, autosync is disabled for this mount", mp)
+	if ctxutil.HasAutoSync(ctx) {
```

Comment:
Wait, you're setting the interactive key above, not the autosync one, no?

Reply:
I propably mixed something up. I need to take a look at it again.
I hope to include meaningful tests as well, as there are none for sync.